src/table/cells/withStyling.jsx

```diff
@@ -1,10 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  srOnly: {
+    clip: 'rect(0 0 0 0)',
+    clipPath: 'inset(50%)',
+    height: '1px',
+    overflow: 'hidden',
+    position: 'absolute',
+    whiteSpace: 'nowrap',
+    width: '1px',
+  },
+});
 
 export default function withStyling(CellComponent) {
   const HOC = (props) => {
-    const { styling } = props;
-    return <CellComponent {...props} className={`${styling.selectedCellClass} sn-table-cell`} style={styling} />;
+    const classes = useStyles();
+    const { styling, column, selState, value } = props;
+
+    const isInSelectionMode = selState.rows.length && Number(column.id.replace(/^\D+/g, '')) === selState.colIdx;
```

Comment:
What is up with the regex? The column index is on the `cell` which is on the props, easier to use that

Reply:
Great suggestion, i was wondering how i can find the columnIdx in the hoc. 
✅ Fixed